qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -568,12 +567,13 @@ def verify_filtered_metrics(metrics):
     def test_perf_stats_stale_metrics_with_multiple_filesystem(self):
         self.mount_a.umount_wait()
         self.mount_b.umount_wait()
+        self.mds_cluster.delete_all_filesystems()
```

Comment:
While this is fine, `class Filesystem` inherits from `class MDSClusterBase`, so the methods are available via `self.fs`. Please revert this hunk.

Reply:
@vshankar use `self.fs` instead?

Reply:
Yes and in L#501.